tc/src/main/java/com/epam/tc/security/SuccessHandler.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.tc.security;
+
+import java.io.IOException;
+import java.util.Set;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.authority.AuthorityUtils;
+import org.springframework.security.web.authentication.AuthenticationSuccessHandler;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SuccessHandler implements AuthenticationSuccessHandler {
```

Comment:
Only authentication success handled here. This should be reflected in class name